boards/xunlong/orangepi_5_ultra_rk3588/orangepi_5_ultra_rk3588_defconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright The Zephyr Project Contributors
+
+# Platform Configuration
+CONFIG_ARM_ARCH_TIMER=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=24000000
```

Comment:
```suggestion
```
Move to soc Kconfig and value must come from dts

Reply:
rk3588 use SCMI to control cpu clock frequency, I just move it to soc Kconfig